### DISCUSSION TITLE
Including v4l2-ctl from v4l-utils?

### DISCUSSION THREAD

**Discussion by andyshinn** [gtxaspec/wz_mini_hacks#106] [Development]
I am running into an issue trying to decode the RTSP stream with FFmpeg that I think it related to https://github.com/mpromonet/v4l2rtspserver/issues/58. The solution there was to set `v4l2-ctl -c repeat_sequence_header=1` but there is no `v4l2-ctl` compiled that I can see. I am not sure if it is even possible to set this option on the Wyze camera.

Possible to include this or confirm / deny if this would be possible to set to get FFmpeg ingestion working?

**Comment by gtxaspec** [comment]
[v4l2-ctl.tar.gz](https://github.com/gtxaspec/wz_mini_hacks/files/8934557/v4l2-ctl.tar.gz)
heres a static binary of v4l2-ctl, though due to the fact that we use the loopback driver, I'm not sure it will work properly.

**Reply by andyshinn** [reply]
Indeed it doesn't seem to do what I was looking for and just segfaults. Thanks for the binary, though!
